src/Components/BookCard/BookCard.test.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import BookCard from './BookCard';
+import { render, screen } from '@testing-library/react';
+import '@testing-library/jest-dom';
+
+describe('BookCard', () => {
```

Comment:
good to be sure this is set up and working, definitely want to make this more robust